build.py

```diff
@@ -72,13 +72,13 @@
 TRITON_VERSION_MAP = {
     "2.40.0dev": (
         "23.11dev",  # triton container
-        "23.09",  # upstream container
+        "23.10",  # upstream container
         "1.16.0",  # ORT
         "2023.0.0",  # ORT OpenVINO
         "2023.0.0",  # Standalone OpenVINO
         "2.4.7",  # DCGM version
         "py310_23.1.0-1",  # Conda version
-        "0.2.1.post1",  # vLLM version
+        "9.1.0.1",  # TRT version for building TRT-LLM backend
```

Comment:
```suggestion
        "0.2.1.post1", # vLLM version
```
It's the vLLm version that we need to keep, and remove the TRT-LLM related ones.

Reply:
@krishung5 - please review again